build.sh

```diff
@@ -299,11 +299,11 @@ if [[ "$WATTSI_RESULT" -ne "0" ]]; then
     echo "The correct line numbers for errors in the $HTML_SOURCE/source file are shown below:"
     echo
     cat $HTML_TEMP/wattsi-output.txt | grep -v '^$' # trim blank lines
```

Comment:
It seems like the actual outputting of the errors is still covered by the `if [[ "$WATTSI_RESULT" -ne "0" ]]` on line 297. So this change doesn't actually start outputting anything, right?